format_checker/pr_checker.py

```diff
@@ -182,5 +182,24 @@ def run_checks_pr(log, commit_range):
         check_forked_project,
         check_tab,
     ]
-    run_checks(filename, pr_data, log, commit_range, checks)
+    run_checks(filename, data, log, commit_range, checks)
+    check_sort(filename, log)
+
+def run_checks_gr(log, commit_range):
```

Comment:
Isn't this the same function as `run_checks_pr` but just applied on a different file?

Reply:
I didn't add an input parameter for this function because in that case I should also change the code for `run_checks_tic_fic` and `run_checks_tso_iso` functions, all the checkers are called from for loop in `main.py` (lines 21-23).

Should I update them and add fileName parameter?

Reply:
Maybe change https://github.com/TestingResearchIllinois/idoft/blob/main/format_checker/main.py#L21 to something like `checks = [lambda: run_checks_pr 'pr-data.csv', lambda: run_checks_pr 'gr-data.csv', run_checks_tic_fic, run_checks_tso_iso]`

Reply:
Thanks for that, I'm quite new to python.
Fixed